main.go

```diff
@@ -2,16 +2,30 @@ package main
 
 import (
 	"./logging"
+	"fmt"
 	"log"
 	"os"
 	"strings"
+	"sync"
+
+	"github.com/hashicorp/go-reap"
 )
 
 func main() {
 	log.SetOutput(os.Stdout)
 	if strings.EqualFold(os.Getenv("DEBUG"), "true") {
 		go logging.StartLogging()
 	}
+	// Get feedback on reaped children and errors.
+	if reap.IsSupported() {
```

Comment:
I thought about removing it as well and your reasoning why there is no need for the check makes sense. Will remove it.